src/actions/link.actions.test.js

```diff
@@ -1,175 +1,176 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
We must update eslint with the following: it's because it doesn't know what is a test file. 
"import/no-extraneous-dependencies": ["error", {"devDependencies": ['*.test.js']}]